solr/server/solr/configsets/_default/conf/solrconfig.xml

```diff
@@ -199,7 +199,7 @@
          Setting The value to true will instruct the underlying Lucene
          IndexWriter to write its debugging info the specified file
       -->
-    <!-- <infoStream file="INFOSTREAM.txt">false</infoStream> -->
+    <infoStream>false</infoStream>
```

Comment:
I see above for other things that we comment out the defaults rather than literally put them here.  I think the general trend should be to *not* have configuration for defaults -- embracing good defaults.

Reply:
I can do this.  I don't mind the verbosity because I might not know what the defaults are, but that would match up with the pattern.  I will check outher configsets.